extensions/cornerstone-dicom-sr/src/commandsModule.ts

```diff
@@ -123,6 +124,47 @@ const commandsModule = (props: withAppTypes) => {
         throw new Error(error.message || 'Error while saving the measurements.');
       }
     },
+
+    /**
+     * Loads measurements using either the given function or if the function is not present
+     * it hydrates and loads the SR for the given display set instance UID and displays
+     * it in the active viewport.
+     * @param loadMeasurementsEventFn the function to call to load the measurements
+     * @param loadMeasurementsEventName the name of the event to fire to load the measurements
+     * @param loadMeasurementsEventArgs arguments for the event; in particular the display set instance UID
+     * @param clearMeasurements optionally clear the measurements prior to hydrating the SR when loadMeasurementsEventFn is not present
+     * @returns
+     */
+    loadMeasurements: ({
```

Comment:
This is more like `loadStructuredReport` since sometimes those SRs are not measurements and just text right? like this study https://viewer-dev.ohif.org/viewer?StudyInstanceUIDs=1.2.276.0.7230010.3.1.2.2155604110.4180.1021041295.21

Reply:
I don't think so, but I am not the expert here. I believe text SRs are ultimately parsed and handled by `OHIFCornerstoneSRContentItem`. This command is strictly for loading (SR) measurements and was extracted from the `OHIFCornerstoneSRMeasurementViewport`